### PR TITLE
Added some NM's

### PR DESCRIPTION
**Pull request by TeoTwawki**
Added Highlander Lizard and Ghillie Dhu, and corrected Haty's job and spell list values. 
(I have NEVER ONCE seen that NM cast any spell at all in retail. EVER.)

As retail as I think I can get them without logging in to retail to double check and calculate their exact str attack and weapon damage. I know its free week, but I ain't got the time.

Highlander Lizard needs his other spawn points in nm_spawn_points.sql - I only have 1 because I lost my notes. If I find them I can update him later, else someone can retail cap them. I forgot if when that idiotic 50 spawn point requirement was change if it was removed altogether or was just reduced, so I duped the line (its going to get changed anyway).

No errors on my test box, and my [GM]Testdummy taru *looked* to be taking appropriate damage fighting these while eyeballing it. All behavior, status, immunity, drops, etc are there.

### CODE REVIEW

File: scripts/zones/Konschtat_Highlands/mobs/Ghillie_Dhu.lua [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobRoam
+-----------------------------------
+
+function onMobRoam(mob)
+    -- Fairly sure he shouldn't be storing up max TP while idle.
+    if (mob:MOD_REGAIN() ~= 0) then
```
Comment (xdemolish) [inline scripts/zones/Konschtat_Highlands/mobs/Ghillie_Dhu.lua]: getMod(MOD_REGAIN)

Comment (TeoTwawki) [inline scripts/zones/Konschtat_Highlands/mobs/Ghillie_Dhu.lua]: Dammit, left wrong copy of file in my trunk. Amending..I did that earlier too so I'm fixing same line TWICE.

**Comment by TeoTwawki** [comment]
Amended.

**Comment by TeoTwawki** [comment]
...forgot the tp drain..Sorry, I've been awake all night I should have waited.

**Comment by TeoTwawki** [comment]
Amended: Fixed error @xdemolish caught, took out the debugging print I accidentally missed and left in, and one other mistake I hope nobody noticed. :blush: Should be all good now.

**Comment by TeoTwawki** [comment]
..GDI. will reopen after I fix. I dunno wtf is wrong with me this morning I keep doing stupid crap.

**Comment by TeoTwawki** [comment]
tardhub wouldn't let me reopen this :rage:

**Comment by xdemolish** [comment]
hover over the button

**Pull request closed without merging**